packages/reports/src/Containers/reports.tsx

```diff
@@ -14,8 +13,33 @@ import {
 import { getSelectedRoute } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
+import { TRoute } from '../Types';
+import { TRootStore } from 'Stores/index';
 import 'Sass/app/modules/reports.scss';
 
+type TList = {
+    value: React.ComponentType | typeof Redirect;
+    default?: boolean;
+    label: string;
+    icon?: string;
+    path?: string;
+};
+
+type TReports = {
+    history: RouteComponentProps['history'];
+    is_logged_in: boolean;
+    is_logging_in: boolean;
+    is_from_derivgo: boolean;
+    is_visible: boolean;
+    location: RouteComponentProps['location'];
+    routeBackInApp: (history: RouteComponentProps['history'], additional_platform_path?: string[]) => void;
+    routes: TRoute[];
+    setTabIndex: (value: number) => void;
+    setVisibilityRealityCheck: (value: number) => void;
+    tab_index: number;
+    toggleReports: (value: boolean) => void;
```

Comment:
@kate-deriv No need to redefine all the types coming from stores again, You can use `TRootStore['ui']['toggleReports']` for example.

Although I **still** suggest using the `useStore` hook from the `@deriv/stores` package so you guys don't need to write the types as it can infer😩

Reply:
Done, thank you!